specification/iothub/resource-manager/Microsoft.Devices/preview/2021-02-01-preview/iothub.json

```diff
@@ -4238,7 +4239,7 @@
           "type": "string",
           "readOnly": true
         },
-        "identityType": {
+        "type": {
```

Comment:
As identity type only allows specific set of values, define this as a enum like below.
 "enum": [
            "SystemAssigned",
            "UserAssigned",
            "SystemAssigned, UserAssigned",
            "None"
          ],
          "x-ms-enum": {
            "name": "ResourceIdentityType",
            "modelAsString": false
          }

For example, refer https://github.com/Azure/azure-rest-api-specs/blob/f862372181b14291ea56a9dd9a8fef4a8a0a132e/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json#L7028

Reply:
Thanks for the feedback, made the changes accordingly.